app/assets/stylesheets/application.scss

```diff
@@ -10,18 +10,14 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require "ui-kit"
  *= require "simplemde.min"
  *= require "trumbowyg.min"
  *= require "editor.md/scss/editormd"
  */
- @import "bourbon";
- @import "neat";
+ @import "../../../vendor/assets/stylesheets/ui-kit";
+ @import "../../../vendor/assets/stylesheets/ui-kit-icons";
 
- @import "colours";
```

Comment:
I see colours.scss has gone ... how will the colours work now ? Are they in UIKit anyway ?